gst/nnstreamer/include/nnstreamer_plugin_api.h

```diff
@@ -379,6 +379,16 @@ gst_tensors_get_caps (GstPad * pad, const GstTensorsConfig * config);
 extern gboolean
 gst_pad_peer_has_tensor_caps (GstPad * pad);
 
+
+/**
+ * @brief Allocating memory block with alignment
```

Comment:
How about adding note that this function is dependent if certain option is not enabled, it does not align?